source/sass/mixins/_typography.scss

```diff
@@ -120,5 +120,45 @@
 @mixin list-style-none() {
   @include padding(0, inline-start);
   // Overflow auto is so the bullet will be still read from a screen reader and will push the bullet off screen
-  overflow: auto;
+  overflow-x: auto;
+}
+
+@mixin list-separator($separator: default) {
+
+  @if $separator == default {
+
+    &:after {
+      content: ",\a0";
+    }
+
+    &:lang(ar):after {
```

Comment:
Tempted to say that we shouldn't test the languages variants, but will delay that until we can generate them from CLDR.